ui/perfherder/alerts/AlertTable.jsx

```diff
@@ -119,6 +123,14 @@ export default class AlertTable extends React.Component {
     this.setState({ filteredAlerts });
   };
 
+  updateAssignee = newAssigneeUsername => {
+    const { alertSummary, updateAlertSummary } = this.props;
+
+    return updateAlertSummary(alertSummary.id, {
```

Comment:
Were you planning to add more commits for error handling in this pr? The `processResponse` helper will make this easy until you decide what to do about whether to adopt the notifications system.

Reply:
Not really. I thought handling the errors directly in the client code from `Assignee` component suffices. I've addressed this now in `AlertTable.updatgeAssignee()`.

However, I'm not sure I did proper error handling in the client code from `Assignee.pressedEnter()`.